cmds/task/fetchers.go

```diff
@@ -58,7 +58,45 @@ func runStatus(credentials *tcclient.Credentials, args []string, out io.Writer,
 	fmt.Fprintln(out, getRunStatusString(s.Status.Runs[runID].State, s.Status.Runs[runID].ReasonResolved))
 	return nil
 }
+// confirmMsg displays confirmation message when --confirm is used
+func confirmMsg(command string, credentials *tcclient.Credentials, args []string, out io.Writer, _ *pflag.FlagSet )(string) {
+	
+	q := makeQueue(credentials)
+	taskID := args[0]
+
+	c, _ := q.Status(taskID)
+	run := c.Status.Runs[len(c.Status.Runs)-1]
+
+	t, _ := q.Task(taskID)
+	
+	fmt.Println(command, t.Metadata.Name ,"taskid:", taskID ,"(state:",run.State,"). Are you sure you want to proceed with given command? (y/N)")
+	var response string
+	fmt.Scanf("%s", &response)
+
+
+	return response
+}
+
+// displayNoopMsg displays details when --noop is used
+func displayNoopMsg(command string, credentials *tcclient.Credentials, args []string, out io.Writer, _ *pflag.FlagSet ) error {
+	q := makeQueue(credentials)
+	taskID := args[0]
 
+	c, _ := q.Status(taskID)
+	run := c.Status.Runs[len(c.Status.Runs)-1]
+
+
+	t, err := q.Task(taskID)
+	if err != nil {
+		return fmt.Errorf("could not get the task %s: %v", taskID, err)
+	}
+
+	fmt.Println(command, t.Metadata.Name ,"taskid:", taskID ,"(state:",run.State,")")
```

Comment:
some whitespace would probably be good here? Perhaps fmt.Printf?

Reply:
Won't work like that :(

Reply:
Sorry, why not?

Reply:
I'm a little dubious that adding whitespace to a message causes "crazy"..